packages/react-components/react-menu/src/components/MenuItemLink/renderMenuItemLink.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { getSlotsNext } from '@fluentui/react-utilities';
+import type { MenuItemLinkState, MenuItemLinkSlots } from './MenuItemLink.types';
+
+/**
+ * Render the final JSX of MenuItemLink
+ */
+export const renderMenuItemLink_unstable = (state: MenuItemLinkState) => {
+  const { slots, slotProps } = getSlotsNext<MenuItemLinkSlots>(state);
+
+  // TODO Add additional slots in the appropriate place
+  return (
+    <slots.root {...slotProps.root}>
+      {slots.checkmark && <slots.checkmark {...slotProps.checkmark} />}
```

Comment:
BTW, can it have `checkmark` at all? How will it be selected?

Reply:
it doesn't have to be selected, but it might need to align with other selectable menuitems